projects/packages/my-jetpack/_inc/components/product-interstitial/index.jsx

```diff
@@ -78,7 +120,7 @@ export function AntiSpamInterstitial() {
  */
 export function BackupInterstitial() {
 	return (
-		<ProductInterstitial slug="backup">
+		<ProductInterstitial slug="backup" installsPlugin={ true }>
```

Comment:
Just a note: I think it shouhd be handled at the hook/data level. If so, we can iterate in follow-up PRs.